src/dbus_fast/aio/message_bus.py

```diff
@@ -318,6 +318,13 @@ async def introspect(
         finally:
             timer_handle.cancel()
 
+    def introspect_sync(
```

Comment:
It looks like this is being added only for test purposes. It would be better to just access `_loop` in the test (if it is actually needed) instead of adding an API that isn't meant to be used. This is what codecov is complaining that isn't covered because it is mocked in the test anyway. So there doesn't seem to be any reason to add this at all.

Reply:
Bah, my browser was being weird and didn't actually find all instance of `introspect_sync` when I was searching for it the first time, so ignore that comment.